variants/happymodel_es900tx/variant.h

```diff
@@ -0,0 +1,56 @@
+// This board do not have extra serial to connect GPS to
+#undef GPS_RX_PIN
+#undef GPS_TX_PIN
+
+// This board do not have easy accessible I2C pins
+#undef I2C_SDA
+#undef I2C_SCL
+
```

Comment:
This just means the firmware will use the standard 21/22 for I2C - if that's not occupied its fine, else please move them out of the way.